src/Haskell/Ide/Engine/Plugin/HieExtras.hs

```diff
@@ -178,6 +182,9 @@ mkPragmaCompl label insertText =
 safeTyThingId :: TyThing -> Maybe Id
 safeTyThingId (AnId i)                    = Just i
 safeTyThingId (AConLike (RealDataCon dc)) = Just $ dataConWrapId dc
+safeTyThingId (ATyCon tyCon)              = case GHC.tyConTyVars tyCon of
```

Comment:
This doesn't look right to me. Should it not get the id of the tycon?

Reply:
Right, since I dont use it all, I should not try to change more code.